GuessingGame/Service.cs

```diff
@@ -3,71 +3,75 @@
 
 namespace GuessingGame
 {
-public class Service
-{
-     private readonly INumberGenerator _numberGenerator;
+    public class GuessingGameService
```

Comment:
For the next exercise, you might want to consider Single Responsibility when naming a class. The GuessingGameService could end up with too many responsibilities and end up being a very large class. Consider if you had to explain what your class does to another developer, what would you tell them the responsibility was?